views/candidaterecord/summary.templ

```diff
@@ -5,42 +5,86 @@ import (
 	"github.com/ugent-library/biblio-backoffice/ctx"
 	"github.com/ugent-library/biblio-backoffice/models"
 	contributorviews "github.com/ugent-library/biblio-backoffice/views/contributor"
+	"github.com/ugent-library/biblio-backoffice/views/util"
+	"math"
+	"time"
 )
 
 type SummaryOpts struct {
-	Thumbnail string
-	Badge     templ.Component
+	Thumbnail   string
+	Badge       templ.Component
+	Info        templ.Component
+	ShowDetails bool
+}
+
+func daysUntilDisappearanceDate(date time.Time) string {
+	disappearanceDate := util.DateOnly(date).AddDate(0, 0, 90)
+	today := util.DateOnly(time.Now())
```

Comment:
Can't you simply `time.Now().Round(0).Date()`? (haven't tested)

Reply:
That doesn't work according to the [docs](https://pkg.go.dev/time#Time.Round) and [my test case](https://goplay.tools/snippet/LVtaF9PHQvE).

You could round to a date using `.Round(time.Hour * 24)` I think but that would round up to the next day after 12:00 which is also not what we want.